### PR TITLE
Web-Tags —> Web

### PR DESCRIPTION
**Pull request by marianbitsov23**
### New Tag Feature
  - Tag selection when signing up
  - Showing tags on user cards in Home/Match page
  - Update/Delete tags in Settings (Edit personal info) page

### CODE REVIEW

**Comment by marianbitsov23** [comment]
One more thing, the logo is changed both in the navbar and the title of the page

File: Web/tues_pairs/src/components/Pages/Home/home.jsx [hunk 1]
```diff
@@ -120,9 +133,12 @@ class UserList extends Component {
           users[i].photoURL = "https://x-treme.com.mt/wp-content/uploads/2014/01/default-team-member.png";
         }
         mappedUsers.push(users[i]);
+        console.log(users[i]);
```
Comment (GGLozanov) [inline Web/tues_pairs/src/components/Pages/Home/home.jsx]: Изтрий това.

**Review by GGLozanov — APPROVED** [review]
LGTM. Само изтрии ако можеш console.log()-а, който съм ти подчертал

**Review by msimov02 — APPROVED** [review]
LGTM!

**Pull request merged**